### DISCUSSION TITLE
Rate limiting for the Vercel Firewall

### DISCUSSION THREAD

**Discussion by FleetAdmiralJakob** [vercel/community#7061] [Feature Question] [answered]
### Question

Lee Robinson promises on the Vercel YT chanel in this video a feature called Rate Limiting.

![image](https://github.com/vercel/community/assets/77451351/e3fc2fe3-b54d-4a15-8220-2ce37d1c3522)

https://www.youtube.com/watch?v=t-U6vTrx_9Y

I can't access this feature on my hobby plan sadly.
![image](https://github.com/vercel/community/assets/77451351/0d493b8d-a349-44fe-b9d9-19dd0d35a25a)

**Comment by leerob** [comment, marked as answer]
Hello! As I mentioned in the video, this is in beta and coming soon 😄 Glad that you are excited for it!

**Reply by FleetAdmiralJakob** [reply]
Oh, sorry my mistake, I should watch the video twice before making such a post here in the future. Thrilled to see this feature, so I dont have to rely on additional services like Upstash for this type if features anymore.

I love this new feature and want to thank the Vercel team for this.

**Comment by KlimentP** [comment]
@leerob hey, any idea when is the rate limiting getting released? I am about to add rate limiting in a project and was wondering whether to implement it the old way with kv/upstash or wait for the release?

[a comment by an automated account is not shown]
